### PR TITLE
unit tests for fixes #6067 - UI inconsistency around root password limit 8 characters

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
Commit 9c30b63 was merged before adding unit tests.

**Comment by isratrade** [comment]
[test]

**Comment by domcleal** [comment]
Thanks, tests are failing though.

**Comment by isratrade** [comment]
@domcleal, very strange, the simple test passed locally. re [test]

File: test/unit/hostgroup_test.rb [hunk 1]
```diff
@@ -287,4 +287,20 @@ def setup_user operation
     assert_equal (hostgroup.environment.puppetclasses - hostgroup.parent_classes).sort, hostgroup.available_puppetclasses.sort
   end
 
+  test "hostgroup root pass can be blank" do
+    hostgroup = FactoryGirl.create(:hostgroup)
+    hostgroup = nil
```
Comment (domcleal) [inline test/unit/hostgroup_test.rb]: I think you meant `hostgroup.root_pass`

**Comment by isratrade** [comment]
@domcleal, thanks. recommitted. The previous strange failure only on Jenkins, not locally, was
`assert hostgroup.root_pass.blank?` so I just decided to make it `hostgroup.root_pass = nil` since the real test is `assert hostgroup.valid?`

**Comment by domcleal** [comment]
Thanks @isratrade, merged as 8e51f43.

**Pull request closed without merging**